pyproject.toml

```diff
@@ -1,2 +1,7 @@
 [build-system]
-requires = ["setuptools", "wheel", "scikit-build"]
+requires = [
+    "setuptools >= 40.8.0",
```

Comment:
This won't work on RHEL 8

Reply:
It works fine because the version of pip used by RHEL 8 totally ignores pyproject.toml.  Versions of pip that do read this file, will use it to download build system requires into a temporary environment for the construction of the wheel.  
And there's no intrinsic requirement that setuptools >= 40.8.0, it's just a requirement for the purposes of a pyproject.toml-based build doing the correct thing.

https://pip.pypa.io/en/stable/reference/pip/#pep-517-and-518-support

![Screenshot from 2020-12-25 13-54-48](https://user-images.githubusercontent.com/2751996/103140794-1929dc80-46b9-11eb-99e4-a0b5ca539c1a.png)